.github/workflows/ci_tests_espidfv5.yml

```diff
@@ -0,0 +1,21 @@
+name: Linux CI Tests
```

Comment:
Can this name be changed so it's differentiable from the existing one?

Reply:
Yes, I changed it.